### PR TITLE
juicefs: add accesskey2 & secretkey2 for mirror fs

### PR DESCRIPTION
**Pull request by zwwhdls**
### Ⅰ. Describe what this PR does
Add access-key2 & secret-key2 in secret for mirror filesystem. Refer to https://juicefs.com/docs/zh/cloud/guide/mirror/

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/juicefs/type.go [hunk 1]
```diff
@@ -45,19 +45,24 @@ type JuiceFS struct {
...
-	Bucket             string `json:"bucket,omitempty"`
-	MetaUrlSecret      string `json:"metaurlSecret,omitempty"`
-	MetaUrlSecretKey   string `json:"metaurlSecretKey,omitempty"`
-	TokenSecret        string `json:"tokenSecret,omitempty"`
-	TokenSecretKey     string `json:"tokenSecretKey,omitempty"`
-	Storage            string `json:"storage,omitempty"`
-	FormatCmd          string `json:"formatCmd,omitempty"`
-	QuotaCmd           string `json:"quotaCmd,omitempty"`
+	Name                string `json:"name"`
+	AccessKeySecret     string `json:"accesskeySecret,omitempty"`
```
Comment (cheyang) [inline pkg/ddc/juicefs/type.go:49]: How about using the following configuration to make it extensible?
```golang
type Configs struct {
    Name      string   `json:"name"`
    Buckets   []Bucket `json:"buckets"`
    // other fields...
}

type Bucket struct {
    Name          string `json:"name"`
    AccessKey     string `json:"accessKey"`
    SecretKey     string `json:"secretKey"`
    // other fields...
}
```

Comment (zwwhdls) [reply pkg/ddc/juicefs/type.go:49]: It's an internal config used by runtime. `configs` is all the options used by `juicefs format` or `juicefs auth`,  its quantity is almost fixed and new additions are rare. So, I think it is meaningless to make it extensible.

Comment (cheyang) [reply pkg/ddc/juicefs/type.go:49]: How about making the dataset api  in the following way extensible and readable? like `bucketname.access-key`?

```yaml
kind: Dataset
metadata:
  name: jfsdemo
spec:
  mounts:
    - name: minio
      mountPoint: "juicefs:///demo"
      options:
        bucket: "bucket1,bucket2"
        storage: "minio"
      encryptOptions:
        - name: metaurl
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: metaurl
        - name: bucket1.access-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: access-key
        - name: bucket1.secret-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: secret-key
        - name: bucket2.access-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: access-key
        - name: bucket2.secret-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: secret-key
```

**Comment by cheyang** [comment]
> ### Ⅰ. Describe what this PR does
> Add access-key2 & secret-key2 in secret for mirror filesystem. Refer to https://juicefs.com/docs/zh/cloud/guide/mirror/
> 
> ### Ⅱ. Does this pull request fix one issue?
> ### Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.
> ### Ⅳ. Describe how to verify it
> ### Ⅴ. Special notes for reviews

Looks like juicefs support multiple mirrors not only 2 mirrors.

File: pkg/ddc/juicefs/type.go (hunk 1 above)
Comment (cheyang) [reply pkg/ddc/juicefs/type.go:49]: I think we don't support this way by default. This is only for  `bucket: "bucket1,bucket2"`

Comment (cheyang) [reply pkg/ddc/juicefs/type.go:49]: If there is only one bucket:

```yaml
apiVersion: data.fluid.io/v1alpha1
kind: Dataset
metadata:
  name: jfsdemo
spec:
  mounts:
    - name: minio
      mountPoint: "juicefs:///"
      options:
        bucket: "http://minio:9000/minio/test"
        storage: "minio"
      encryptOptions:
        - name: metaurl
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: metaurl
        - name: access-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: access-key
        - name: secret-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: secret-key
```

If there are more than one bucket:

```yaml
kind: Dataset
metadata:
  name: jfsdemo
spec:
  mounts:
    - name: minio
      mountPoint: "juicefs:///demo"
      options:
        bucket: "bucket1,bucket2"
        storage: "minio"
      encryptOptions:
        - name: metaurl
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: metaurl
        - name: bucket1.access-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: access-key
        - name: bucket1.secret-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: secret-key
        - name: bucket2.access-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: access-key
        - name: bucket2.secret-key
          valueFrom:
            secretKeyRef:
              name: jfs-secret
              key: secret-key
```

**Pull request closed without merging**